tests/utils_tests.py

```diff
@@ -32,8 +32,11 @@
 import lsst.daf.base as dafBase
 import lsst.geom
 
+# # Create an instance of random generator with fixed seed.
+# rng = np.random.default_rng(1234)
```

Comment:
Did you intend for this to be commented out?

Reply:
No sorry. Thanks for the catch